src/Speech/SpeechClient.php

```diff
@@ -188,6 +188,13 @@ public function __construct(array $config = [])
      *           to favor specific words and phrases in the results. Please see
      *           [SpeechContext](https://cloud.google.com/speech/reference/rest/v1/RecognitionConfig#SpeechContext)
      *           for more information.
+     *     @type bool $enableWordTimeOffsets If set to `true`, a list of
+     *           `word_time_offsets` are returned with the top alternative. If
```

Comment:
Done. I was on the fence about putting anything in the result class - I have added comments now.